mac

```diff
@@ -128,9 +131,8 @@ brew install rbenv
 rbenv init
 echo 'eval "$(rbenv init -)"' >>~/.bash_profile
 
-
 fancy_echo "Installing latest Ruby..."
```

Comment:
we should probably make this explicitly say which ruby it's installing.  Cause this is very much not the latest.

Reply:
Good point, this will be consistent with the Node installation message, too! Addressed.